types.ts

```diff
@@ -791,8 +791,12 @@ export interface Format {
    */
   instructions?: string;
   /**
+   * to be deprecated
+   *
    * A list of regular expressions that match URLs that the formula implementing this format
    * is capable of handling. As described in {@link Format}, this is a discovery mechanism.
+   *
+   * This will be replaced by the `matchers` property in {@link makeFormula} in a future version.
```

Comment:
When pasting a URL into a table, and it matches a *formula's* regex, is a column format still required? Options:
1) Yes, a column format is required to have the table-pasting entrypoint.
2) No, we automatically create a "column format" for every formula, and the only purpose of addColumnFormat is to customize it.
3) No, but the column only becomes a normal formula column, not a pack-typed column.

I'm inclined towards #3.

Reply:
hmm honestly i think 3 is a little too aggressive, esp becuase it doesn't apply at the cell level.

i think 1 is better to have it be explicit

Reply:
I'm not necessarily opposed to #1, but just in case you don't know - the pack magic for URL pasting in tables only works on a completely blank column - if the format of the column has been set at all (e.g., by some content being put into it), then the pack magic won't trigger. That's why #3 didn't seem aggressive to me.

Reply:
actually im not really sure what you mean by #3, how does it become a normal formula column and not a pack-typed column? it needs the pack column in order to have the magic of you being able to type in a new value at each cell

and yeah even knowing the constraints on the behavior i think i prefer being explicit here

Reply:
Oh, yes, I wasn't thinking through that carefully enough. I could imagine a popup promotion offering to make a new additional column that uses the URL as input, but you're totally right that we can never do auto-transformation magic in this case.